core/arch/arm/mm/tee_mmu.c

```diff
@@ -900,10 +900,16 @@ TEE_Result tee_mmu_map_param(struct user_mode_ctx *uctx,
 		mem[n].size = ROUNDUP(phys_offs + param->u[n].mem.offs -
 				      mem[n].offs + param->u[n].mem.size,
 				      CORE_MMU_USER_PARAM_SIZE);
+		/*
+		 * For size 0 (raw pointer parameter), add minimum size
+		 * value to allow address to be mapped
+		 */
+		if (!mem[n].size)
```

Comment:
`if (mem[n].mobj && !mem[n].size)` ?

Reply:
No only the size must be checked.

Reply:
I guess if `mem[n].mobj` is NULL then `mem[n].size` is meaningless at TA entry. However, for consistency, it looks strange to force a non-null size value for a null memory reference.